cfn-resources/cloud-backup-restore-jobs/README.md

```diff
@@ -0,0 +1,62 @@
+# MongoDB::Atlas::CloudBackUpRestoreJobs
+
+## Description
+This resource allows you to create, cancel, get one or list all cloud backup restore jobs.
```

Comment:
```suggestion
Returns, starts, and cancels Cloud Backup restore jobs.
```
Use Open API Doc deflations as much as possible: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Cloud-Backup-Restore-Jobs

Reply:
This issue is fixed